app/src/main/res/values/strings.xml

```diff
@@ -69,6 +69,8 @@
 to your home internet.</string>
   <string name="connected_to_home_assistant">Connected to Home Assistant</string>
   <string name="create_template">Create Template</string>
+  <string name="database_migration_failed">Database migration failed, all sensor data has been reset. Please re-enable your sensors.</string>
```

Comment:
We also store widget data in the DB, should warn them about those needing to be recreated as well.